Tensile/Code.py

```diff
@@ -20,7 +20,7 @@
 ################################################################################
 
 from __future__ import print_function
-from .Common import globalParameters, printExit, printWarning
+from .Common import globalParameters, printExit
```

Comment:
The pyflakes step in Jenkins can't cause a build failure.  I ran pyflakes as a pre-checkin step; this file has lots of variables that are assigned but never used.